src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationIntTest.kt

```diff
@@ -500,6 +557,62 @@ class ContactPersonMigrationIntTest : SqsIntegrationTestBase() {
           assertThat(extension).isEqualTo("ext 5555")
         }
       }
+
+      @Test
+      fun `will send employments to DPS`() {
+        val person = requests.find { it.personId == 1000L } ?: throw AssertionError("Request not found")
+        assertThat(person.employments).hasSize(1)
+        with(person.employments!![0]) {
+          assertThat(sequence).isEqualTo(1)
+          assertThat(corporate.id).isEqualTo(120)
+          assertThat(corporate.name).isEqualTo("Police")
+          assertThat(active).isTrue()
+          assertThat(createUsername).isEqualTo("ADJUA.BEEK")
+          assertThat(createDateTime).isEqualTo(LocalDateTime.parse("2022-02-02T10:23"))
+          assertThat(modifyUsername).isEqualTo("ADJUA.MENSAH")
+          assertThat(modifyDateTime).isEqualTo(LocalDateTime.parse("2024-02-02T10:23"))
```

Comment:
I thought we weren't sending audit data anymore?

Reply:
Contacts haven't changed this yet - unlike CSIP